x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/action/PutAutoFollowPatternRequestTests.java

```diff
@@ -63,6 +64,7 @@ protected boolean supportsUnknownFields() {
         if (randomBoolean()) {
             request.setFollowIndexNamePattern(randomAlphaOfLength(4));
         }
+        request.setLeaderIndexExclusionPatterns(Arrays.asList(generateRandomStringArray(4, 4, false)));
```

Comment:
Can we do this randomly only?

Also, would be nice to generate a list of random length.